lib/SQLiteCpp

```diff
@@ -1 +1 @@
-Subproject commit 7e9983b00fb44c2e569ba54bfee6f548496d7f61
+Subproject commit af78d59349fd013341746b345961e3f19ecae60d
```

Comment:
@Seadragon91 wouldn't it still use the version from af78d59349fd013341746b345961e3f19ecae60d?